api/caching/tasks.py

```diff
@@ -0,0 +1,40 @@
+import urlparse
+
+import celery
+import requests
+from celery.utils.log import get_task_logger
+from django.conf import settings
+
+from framework.tasks import app as celery_app
+
+logger = get_task_logger(__name__)
+
+class VarnishTask(celery.Task):
+    abstract = True
+    max_retries = 5
+
+def get_varnish_servers():
+    #  TODO: this should get the varnish servers from HAProxy or a setting
+    return settings.VARNISH_SERVERS
+
+@celery_app.task(base=VarnishTask, name='caching_tasks.ban_url')
+# @logged('ban_url')
```

Comment:
I generally think of commented-out code as a bad smell. Is it common practice for OSF?